L/LLVM_AMDGPU/build_tarballs.jl

```diff
@@ -0,0 +1,59 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "LLVM_AMDGPU"
+version = v"3.8.0"
+
+# Collection of sources required to build
+sources = [
+    ArchiveSource("https://github.com/RadeonOpenCompute/llvm-project/archive/rocm-$(version).tar.gz",
+                  "93a28464a4d0c1c9f4ba55e473e5d1cde4c5c0e6d087ec8a0a3aef1f5f5208e8")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/llvm-project*/
+
+mkdir build && cd build
+cmake -DCMAKE_BUILD_TYPE=Release \
+      -DCMAKE_INSTALL_PREFIX=${prefix} \
+      -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+      -DLLVM_ENABLE_PROJECTS="clang;lld;clang-tools-extra;compiler-rt" \
+      -DLLVM_ENABLE_ASSERTIONS=ON \
+      -DBUILD_SHARED_LIBS=OFF \
+      ../llvm
+make -j${nproc}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+# The rest of the ROCm stack only seem to support 64bit Linux:
+platforms = [
+    Linux(:x86_64, libc=:glibc),
+    Linux(:x86_64, libc=:musl),
```

Comment:
Same comment as https://github.com/JuliaPackaging/Yggdrasil/pull/1735#discussion_r499186246
```suggestion
    Platform("x86_64", "linux"; libc="glibc"),
    Platform("x86_64", "linux"; libc="musl"),
```